Git Testing/Scenes/Listing/ListViewModel.swift

```diff
@@ -65,6 +51,6 @@ extension ListViewModel: UITableViewDataSource{
 extension ListViewModel:UITableViewDelegate{
     
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        
+        self.didSelectCellClosure?(self.pictures[indexPath.row])
     }
 }
```

Comment:
Update this viewModel to remove the tableview methods and one more thing I noticed in that viewModel is that you are passing pictures from outside. I think you need to fetch them from api, Isn't it ?